frontend/packages/dev-console/src/components/import/app/ApplicationSelector.tsx

```diff
@@ -34,16 +39,15 @@ const ApplicationSelector: React.FC<ApplicationSelectorProps> = ({ namespace })
 
   const handleOnLoad = (items: { [key: string]: string }) => {
     const isEmptyItems = _.isEmpty(items);
-    setIsZeroApplication(isEmptyItems);
+    setShowAppDropdown(!isEmptyItems);
     if (isEmptyItems) {
       setFieldValue('application.selectedKey', CREATE_APPLICATION_KEY);
       setFieldValue('application.name', '');
     }
   };
```

Comment:
I'd replace this whole function with this (it's more-or-less just more explicit variables). I am aware you didn't name `items` but it's easier to read code you're unfamiliar with if you don't use vague names, so I changed it to be more explicit.
```
  const handleOnLoad = (applicationList: { [key: string]: string }) => {
    const noApplications = _.isEmpty(applicationList);
    setApplicationsAvailable(!noApplications);
    if (noApps) {
      setFieldValue('application.selectedKey', CREATE_APPLICATION_KEY);
      setFieldValue('application.name', '');
    }
  };
```